CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 StorIO Change Log
 ==========
 
+## Version 1.12.0
+
+_2016_12_7_
+
+* Support for **AutoValue** and **Kotlin** in StorIOSQLiteProcessor and StorIOContentResolverProcessor. Many thanks to [@geralt-encore](https://github.com/geralt-encore)!
+* DefaultStorIOSQLite combines affected tables from pending changes. **After transaction end will be send single notification, instead of multiple calls on every change**.
```

Comment:
"now combines".  "After the end of transaction DefaultStorIOSQLite will send only one notification instead of multiple for every change." This should not be breaking behavior since `get()` operation is ready to consume such combined changes events.